lua/fpp/server/core.lua

```diff
@@ -130,6 +130,14 @@ hook.Add("PlayerSpawnedSWEP", "FPP.Spawn.SWEP", function(ply, ent)
     ent:CPPISetOwner(ply)
 end)
 
+hook.Add("WeaponEquip", "FPP.Spawn.Weapon", function(weapon)
+    timer.Simple(0, function()
```

Comment:
Why is there a need for the timer?

Reply:
![image](https://github.com/user-attachments/assets/2055cc20-f178-4668-9dcb-d296e76454c8)

Reply:
Could you add a quick comment with something like "this timer is here because Get Owner returns NULL until the next frame"?

Thanks!